### PR TITLE
Improve documents on how to use new docker images

### PR DESCRIPTION
**Pull request by zasdfgbnm**
The original list

```yaml
# * cimodel/data/pytorch_build_definitions.py
# * cimodel/data/caffe2_build_definitions.py
```

is not complete. We should suggest using grep to find where to change because the CI files are changing.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,15 +1,21 @@
 # WARNING: DO NOT EDIT THIS FILE DIRECTLY!!!
 # See the README.md in this directory.
 
-# IMPORTANT: To update Docker image version, please first update
-# https://github.com/pytorch/ossci-job-dsl/blob/master/src/main/groovy/ossci/pytorch/DockerVersion.groovy and
```
Comment (zasdfgbnm) [inline .circleci/config.yml]: Do we still need to modify this file? This file seems to be missing the docker version `a8006f9a-272d-4478-b137-d121c6f05c83` we are currently using.

**Comment by zasdfgbnm** [comment]
I actually have a question: Although we do have weekly build to update the docker, the docker version seems to be hardcoded in lots of places. Does this mean, the weekly job for docker build would push a new docker image, but it will never be used because we are not updating these versions there?

**Comment by ezyang** [comment]
> I actually have a question: Although we do have weekly build to update the docker, the docker version seems to be hardcoded in lots of places. Does this mean, the weekly job for docker build would push a new docker image, but it will never be used because we are not updating these versions there?

Yes, we have to manually land a PR to update to the latest Docker version. The weekly builds exist more to sure that the Docker build process is continuing to work.

File: .circleci/config.yml (hunk 1 above)
Comment (zasdfgbnm) [reply .circleci/config.yml]: @ezyang I just updated this PR to remove groovy related text.

**Comment by kostmo** [comment]
## :pill: CircleCI build failures summary and remediations
As of commit 428044aa:


**None of the build failures appear to be your fault**.

* **8/8** recognized as flaky :snowflake:
    * Re-run these jobs?


## Detailed failure analysis

One may explore the probable reasons each build failed interactively [on the Dr. CI website](https://dr.pytorch.org/commit-details.html?sha1=428044aa92d922b659cd3b851fe8284ff509551e).

### :snowflake: 8 failures recognized as flaky
The following build failures have been detected as flaky and **may not be your fault**:
#### [See CircleCI build](https://circleci.com/gh/pytorch/pytorch/4256884) pytorch_linux_xenial_py2_7_9_build (1/8)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=86577969) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=86577969)) :snowflake:
<details>
<summary>
<code>Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py2.7.9:a8006f9a-272d-4478-b137-d121c6f05c83 not found</code>
</summary>

```
DOCKER_IMAGE: 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py2.7.9:a8006f9a-272d-4478-b137-d121c6f05c83 
Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py2.7.9:a8006f9a-272d-4478-b137-d121c6f05c83 not found 
```

</details>

#### [See CircleCI build](https://circleci.com/gh/pytorch/pytorch/4256877) pytorch_linux_xenial_py3_clang5_android_ndk_r19c_x86_32_build (2/8)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=86577982) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=86577982)) :snowflake:
<details>
<summary>
<code>Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3-clang5-android-ndk-r19c:a8006f9a-272d-4478-b137-d121c6f05c83 not found</code>
</summary>

```
DOCKER_IMAGE: 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3-clang5-android-ndk-r19c:a8006f9a-272d-4478-b137-d121c6f05c83 
Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3-clang5-android-ndk-r19c:a8006f9a-272d-4478-b137-d121c6f05c83 not found 
```

</details>

#### [See CircleCI build](https://circleci.com/gh/pytorch/pytorch/4256880) pytorch_linux_xenial_py3_clang5_mobile_build (3/8)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=86577983) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=86577983)) :snowflake:
<details>
<summary>
<code>Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3-clang5-asan:a8006f9a-272d-4478-b137-d121c6f05c83 not found</code>
</summary>

```
DOCKER_IMAGE: 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3-clang5-asan:a8006f9a-272d-4478-b137-d121c6f05c83 
Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3-clang5-asan:a8006f9a-272d-4478-b137-d121c6f05c83 not found 
```

</details>

#### [See CircleCI build](https://circleci.com/gh/pytorch/pytorch/4256887) pytorch_linux_xenial_py3_clang5_android_ndk_r19c_mobile_code_analysis (4/8)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=86578040) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=86578040)) :snowflake:
<details>
<summary>
<code>Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3-clang5-android-ndk-r19c:a8006f9a-272d-4478-b137-d121c6f05c83 not found</code>
</summary>

```
DOCKER_IMAGE: 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3-clang5-android-ndk-r19c:a8006f9a-272d-4478-b137-d121c6f05c83 
Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3-clang5-android-ndk-r19c:a8006f9a-272d-4478-b137-d121c6f05c83 not found 
```

</details>

#### [See CircleCI build](https://circleci.com/gh/pytorch/pytorch/4256898) pytorch_linux_xenial_py3_6_gcc5_4_build (5/8)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=86578071) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=86578071)) :snowflake:
<details>
<summary>
<code>Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3.6-gcc5.4:a8006f9a-272d-4478-b137-d121c6f05c83 not found</code>
</summary>

```
DOCKER_IMAGE: 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3.6-gcc5.4:a8006f9a-272d-4478-b137-d121c6f05c83 
Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3.6-gcc5.4:a8006f9a-272d-4478-b137-d121c6f05c83 not found 
```

</details>

#### [See CircleCI build](https://circleci.com/gh/pytorch/pytorch/4256895) pytorch_libtorch_linux_xenial_cuda9_cudnn7_py3_build (6/8)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=86578104) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=86578104)) :snowflake:
<details>
<summary>
<code>Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-cuda9-cudnn7-py3:a8006f9a-272d-4478-b137-d121c6f05c83 not found</code>
</summary>

```
DOCKER_IMAGE: 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-cuda9-cudnn7-py3:a8006f9a-272d-4478-b137-d121c6f05c83 
Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-cuda9-cudnn7-py3:a8006f9a-272d-4478-b137-d121c6f05c83 not found 
```

</details>

#### [See CircleCI build](https://circleci.com/gh/pytorch/pytorch/4256888) pytorch_xla_linux_xenial_py3_6_clang7_build (7/8)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=86578175) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=86578175)) :snowflake:
<details>
<summary>
<code>Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3.6-clang7:a8006f9a-272d-4478-b137-d121c6f05c83 not found</code>
</summary>

```
DOCKER_IMAGE: 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3.6-clang7:a8006f9a-272d-4478-b137-d121c6f05c83 
Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-py3.6-clang7:a8006f9a-272d-4478-b137-d121c6f05c83 not found 
```

</details>

#### [See CircleCI build](https://circleci.com/gh/pytorch/pytorch/4256896) pytorch_linux_xenial_cuda9_cudnn7_py3_build (8/8)
**Step:** "Build" ([full log](https://dr.pytorch.org/api/view-log-full?build_id=86578194) | [pattern match details](https://dr.pytorch.org/build-details.html?build_id=86578194)) :snowflake:
<details>
<summary>
<code>Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-cuda9-cudnn7-py3:a8006f9a-272d-4478-b137-d121c6f05c83 not found</code>
</summary>

```
DOCKER_IMAGE: 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-cuda9-cudnn7-py3:a8006f9a-272d-4478-b137-d121c6f05c83 
Error response from daemon: manifest for 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/pytorch-linux-xenial-cuda9-cudnn7-py3:a8006f9a-272d-4478-b137-d121c6f05c83 not found 
```

</details>



---

<details><summary>This comment was automatically generated by <a href="https://github.com/kostmo/circleci-failure-tracker/tree/master/docs/from-pull-request-comment">Dr. CI</a> (expand for details).</summary>Follow <a href="https://dr.pytorch.org/admin/comments-opt-out.html">this link to opt-out</a> of these comments for your Pull Requests.

Please report bugs/suggestions on the <a href="https://github.com/kostmo/circleci-failure-tracker/issues">GitHub issue tracker</a>.
</details>

**Comment by ezyang** [comment]
Oops, I forgot that this PR existed, and merged another version of it. I think this is obsolete now, sorry about that, and thanks for sending this in.

**Pull request closed without merging**